legal-api/src/legal_api/models/review.py

```diff
@@ -79,15 +80,48 @@ def get_review(cls, filing_id) -> Review:
         return review
 
     @classmethod
-    def get_paginated_reviews(cls, page, limit):
-        """Return paginated reviews."""
+    def get_paginated_reviews(cls, review_filter, mapped_sort_by_column):
+        """Return filtered, sorted and paginated reviews."""
         query = db.session.query(Review, Filing.effective_date). \
-            join(Filing, Filing.id == Review.filing_id). \
-            order_by(Review.creation_date.asc())
-
-        pagination = query.paginate(per_page=limit, page=page)
+            join(Filing, Filing.id == Review.filing_id)
+
+        if review_filter.start_date:
+            start_date_utc = LegislationDatetime.as_utc_timezone_from_legislation_date_str(review_filter.start_date)
+            query = query.filter(Review.submission_date >= start_date_utc)
+        if review_filter.end_date:
+            end_date_utc = LegislationDatetime.as_utc_timezone_from_legislation_date_str(review_filter.end_date)
+            query = query.filter(Review.submission_date <= end_date_utc)
```

Comment:
I am not sure about the date conversion. (https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/models/review.py#L120) in json we just provide a date (without conversion), does the UI provide a date in legislative timezone? @severinbeauvais

Reply:
@ArwenQin ^^^ What date-time format do you expect in the UI?

Reply:
I expect '2024-07-01T19:00:00.000+00:00'

Reply:
In json response its in the isoformat (UTC). I believe you convert that before showing in the table. When user select a date from the filter do you convert that to UTC before calling API?

Reply:
This what I see in network tab when I select date range:
https://legal-api-dev.apps.silver.devops.gov.bc.ca/api/v2/admin/reviews?startDate=2024-08-01&endDate=2024-08-28&nrNumber=&identifier=&completingParty=&status=CHANGE_REQUESTED&status=RESUBMITTED&status=AWAITING_REVIEW&sortBy=submissionDate&sortDesc=false&page=1&limit=5

startDate=2024-08-01
endDate=2024-08-28

Reply:
I convert it to 'yyyy-mm-dd' when show it in the table; and when user selects a date from the filter, I pass in  'yyyy-mm-dd' format, date only and without time

Reply:
> In json response its in the isoformat (UTC). I believe you convert that before showing in the table.

Yes.

> When user select a date from the filter do you convert that to UTC before calling API?

No. It's just a YYYY-MM-DD date (assumed to be Pacific timezone).

Reply:
Ketaki, if I understand this correctly, you are converting from a "legislation date string" (date only in Pacific tz) to a UTC date-time string for the db query. This sounds right to me.

Reply:
Yes that is correct